### PR TITLE
 return self for container op initialization methods to allow chaining the construction

### PR DESCRIPTION
**Pull request by IronPan**
Please see test cases for examples how this would look like.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/pipelines/313)

### CODE REVIEW

File: sdk/python/tests/compiler/compiler_tests.py [hunk 1]
```diff
@@ -36,11 +36,11 @@ def test_operator_to_template(self):
       msg2 = dsl.PipelineParam('msg2', value='value2')
       op = dsl.ContainerOp(name='echo', image='image', command=['sh', '-c'],
                            arguments=['echo %s %s | tee /tmp/message.txt' % (msg1, msg2)],
-                           file_outputs={'merged': '/tmp/message.txt'})
-      op.add_volume_mount(k8s_client.V1VolumeMount(
+                           file_outputs={'merged': '/tmp/message.txt'}). \
```
Comment (Ark-kun) [inline sdk/python/tests/compiler/compiler_tests.py]: Nit: It might look better if you move the dot to the next line (here and everywhere).

Comment (IronPan) [reply sdk/python/tests/compiler/compiler_tests.py]: done

**Pull request merged**